pkg/apis/servicecatalog/types.go

```diff
@@ -118,6 +118,9 @@ type ServiceBrokerStatus struct {
 	// was last processed by the controller. The reconciled generation is updated
 	// even if the controller failed to process the spec.
 	ReconciledGeneration int64
+
+	// OperationStartTime is the time at which the current operation began
+	OperationStartTime *metav1.Time `json:"operationStartTime,omitempty"`
```

Comment:
The type definition in `pkg/apis/servicecatalog/types.go` do not include the JSON marshaling information. That should only be put in `pkg/apis/servicecatalog/v1alpha1/types.go`.